docs/jsonapi/create_expression.json

```diff
@@ -6,7 +6,11 @@
       "langcode": "en",
       "field_publication_date": "2019-03-11",
       "field_publication_name": "Newest expression: Publication name",
-      "field_frbr_uri": "/akn/za/act/1993/31/eng@2019-03-11"
+      "field_frbr_uri": "/akn/za/act/1993/31/eng@2019-03-11",
+      "field_content": {
+        "value": "<p><b>Paragraph #1</b></p><p>Paragraph #2</p>",
```

Comment:
@stefanbutura can you open this in the editor and re-save? AFAIK CKEditor requires the whole content to be rendered into an enclosing <p> ... </p>?

Reply:
After re-save, the content remains unchanged. CKEditor wraps text in <p> tags if it is not already in a <p> tag.